pkg/sdwan/vmanage/operation.go

```diff
@@ -384,6 +411,25 @@ func (v *Client) addApplications(ctx context.Context, ops []*sdwan.Operation, lo
 	// listIDs is map that associates custom app name -> policy application list ID
 	listIDs := map[string]string{}
 	for _, customApp := range customApplications {
+		// Does it already exist?
+		appList, err := v.PolicyApplicationsList().
+			GetApplicationListByName(ctx, customApp.Name)
+		if err != nil {
+			if !errors.Is(err, sdwan.ErrNotFound) {
+				log.Err(err).
+					Str("current-app", customApp.Name).
+					Msg("error while checking if policy exists, next operations may fail")
```

Comment:
```suggestion
					Msg("error while checking if policy application list exists, next operations may fail")
```

Reply:
Should the message be like this? As per the name used in the message below.

Reply:
Thanks! Commited the change